core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dataprocModels.scala

```diff
@@ -7,17 +7,17 @@ import enumeratum.{Enum, EnumEntry}
 import org.broadinstitute.dsde.workbench.google2.{InstanceName, ZoneName}
 import org.broadinstitute.dsde.workbench.model.google.GoogleProject
 
-/** Dataproc Instance Status
+/** Google Compute Instance Status
  *  See: https://cloud.google.com/compute/docs/instances/checking-instance-status */
 sealed trait InstanceStatus extends EnumEntry
```

Comment:
Could we move `InstanceStatus` to `gceModels.scala`? (I think I deleted the file but we could re-add it). I feel like it doesn't belong in `dataprocModels`.

Reply:
After scrolling through, I also wondered if `statuses` deserves their own file so they're all in one place; i.e. `statuses.scala` could contain all of the Leo, Dataproc and GCE statuses.

Reply:
I'd prefer not to have a separate `statuses.scala` file. we have separate `runtimeModels`, `gceModels`, `dataprocModels` files already. It seems pretty arbitrary to separate statuses out into a separate file that contain kinda overlaps with all the other 3 files.